src/index.ts

```diff
@@ -32,6 +32,18 @@ export const handleBase64Json = (tokenUri: string): Reason[] => {
   return reasons;
 };
 
+export const handleIpfs = async (tokenUri: string): Promise<Reason[]> => {
+  let reasons: Reason[] = [Reasons.tokenUriIsIpfs];
+
+  let res: Metadata = await axios.get(tokenUri);
+
+  if (res.image && isTokenUriIpfs(res.image)) {
```

Comment:
What is there is no res.image? Is that an error or a /shrug?

This would be nice to have a non-counted issue or _warning_